### ISSUE TITLE
Implement persons deduplication

### ISSUE THREAD

**Issue by roll** [opentrials/opentrials#185]
Entities with dedup based on unique slug:
- condition
- intervention
- location
- organisation

Entities with dedup can't be based on unique slug:
- trial
- person (not implemented for now)

**Comment by roll** [comment]
@vitorbaptista 
May be we need hide the same names in advanced search drop-down? Because this problem will not be solved on 100% even after dedup.

**Comment by vitorbaptista** [comment]
@roll Agreed, specially because there might be cases with two people with the same name. There's probably some configuration for ElasticSearch to return only unique results.
